src/Tests/dotnet-new.Tests/Approvals/DotnetNewInstantiateTests.CannotInstantiateTemplateWithUnknownLanguage.verified.txt

```diff
@@ -1,8 +1,8 @@
 ﻿No templates found matching: 'console', --language='D#'.
 Allowed values for '--language' option are: 'C#', 'F#', 'VB'.
 
-To list installed templates, run:
-   dotnet new list
+To list installed templates similar to 'console', run:
```

Comment:
@baronfel do we need these examples in this case.

To me, this sounds sufficient:

```
﻿No templates found matching: 'console', --language='D#'.
Allowed values for '--language' option are: 'C#', 'F#', 'VB'.
```

Reply:
I think what you're asking is "do we need to show list/search suggestions when the reason a template wasn't found was due to missing/ambiguous argument values?" is that correct? 

If so, yes I agree with you - in these cases we've been able to identify at least the template group the user intends to use, and we already show a useful error message (in this case showing the allowed values for the incorrect parameter), so there's no need to try and drive the user towards list or search.